src/RepublicanSilence/AdherentZone/CommitteeHostZoneRetriever.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace AppBundle\RepublicanSilence\AdherentZone;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\CommitteeMembership;
+use Symfony\Component\HttpFoundation\Request;
+
+class CommitteeHostZoneRetriever implements AdherentZoneRetrieverInterface
+{
+    public function getAdherentZone(Adherent $adherent, Request $request): array
```

Comment:
It would be more flexible and easier to test if we only pass the slug value directly. instead of the whole Request object.